vignettes/datatable-importing.Rmd

```diff
@@ -122,6 +124,12 @@ The case for `:=` is slightly different, because `:=` is interpreted as a functi
 `:=` = function(...) NULL
 ```
 
+The case for `data.table`'s special symbols (`.SD`, `.BY`, `.N`, `.I`, `.GRP`, and `.EACHI`; see `?.N`) is slightly different. We recommend you import whichever of these values you use from `data.table`'s namespace to protect against any issues arising from the unlikely scenario that we change the exported value of these in the future, e.g. if you want to use both `.N` and `.I`, a minimal `NAMESPACE` would have:
```

Comment:
mention it is only relevant when not `import(data.table)`

Reply:
updated, please have a look